apps/backend/migrations/20210615141642-add_api_token.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.createTable('ApiKeys', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT
+      },
+      userId: {
```

Comment:
This userId is not a foreign key. It needs to reference the users table. Right now its just a number.

Look at `migrations/20201201195932-add_user_id_to_evaluations.js` for an example